internal/fs/fs.go

```diff
@@ -7,15 +7,28 @@ import (
 )
 
 type FS struct {
-	EmbedFS embed.FS
+	Core           embed.FS
+	InjectOverride *embed.FS
 }
 
 func (f *FS) Open(path string) (io.ReadCloser, error) {
-	return f.EmbedFS.Open(path)
+	if f.InjectOverride != nil {
+		if file, err := f.InjectOverride.Open(path); err == nil {
+			return file, nil
+		}
```

Comment:
Rather than silently ignoring the error from InjectOverride.Funcs, can you please check them?
They likely return an fs.ErrNotExist. If there is any other error for any reason, throw the error.